src/_pytest/assertion/util.py

```diff
@@ -146,7 +148,7 @@ def assertrepr_compare(config, op, left, right):
                     type_fn = (isdatacls, isattrs)
                     explanation = _compare_eq_cls(left, right, verbose, type_fn)
                 elif verbose > 0:
-                    explanation = _compare_eq_verbose(left, right)
+                    return expl + "\n~" + "\n~".join(_compare_eq_verbose(left, right))
```

Comment:
Sorry, I don't get the `"\n~"` part...? 🤔

Reply:
It mimics https://github.com/blueyed/pytest/blob/1543a16330a5be10022505429b66608e7ee0ed19/src/_pytest/assertion/__init__.py#L139 (which is skipped for strings).
It's the "mini language" used with https://github.com/blueyed/pytest/blob/1543a16330a5be10022505429b66608e7ee0ed19/src/_pytest/assertion/util.py#L59-L91.